index.html

```diff
@@ -5,217 +5,219 @@
+                         </ul>
+                         <p class="underline">
+                            A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                         </p>
+                         <ul class="lang">
+                           <li>html</li>
+                           <li>Rubay And Rails</li>
+                           <li>css</li>
+                           <li>javascript</li>
+                           </ul>
+                             <button type="button">See Project</button>
+                        </div>
+                        <img class="works-image" src="imgs/Snapshoot Portfolio3.png" alt="card 1">
+                        </div>
+                        
+                    </div>
 
-        <div class="card4">
-            <img src="image/card4.png" alt="">
-        <h2>Multi-Post</h2>
-        <h3>Stories</h3> 
-          <ul>
-           <li>CANOPY</li>
-         <li><img src="image/image4.png" alt="count1"></li>
-            <li>Back End Dev</li>
-            <li><img src="image/image4.png" alt="count2"></li>
-            <li>2015</li>
-        </ul>
-        <p>
-            A daily selecion of privately personalized reads; no accounts or sign up required.
-        </p>
-         <ul class="lang">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-         </ul>
-         <button type="button">See Project</button>
-        </div>
-    
     </section>
 
-    <section>
-    
-    <div class="about">
-        <h2>About </h2>
-        <h3> MySelf</h3>
-        <p class="intro">
-            Hello I am a software developer! I can help you build a product ,feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <div class="contact">
-            <h4>LET'S  CONNECT</h4>
-             </div>
-             <div class="icons">
-        <ul>
-          <li><img src="image/image6.png" alt="twitter"></li>
-          <li><img src="image/image7.png" alt="twitter"></li>
-          <li><img src="image/image8.png" alt="twitter"></li>
-           <li><img src="image/image9.png" alt="twitter"></li>
-          <li><img src="image/imge5.png" alt="twitter"></li>
-        </ul>
-        </div>
-          
-          <button type="button">Get My Resume</button>
+    <section class="about">
+        <div class="about-card">
+        <div class="introduction-2">
+            <h1 class="title">About <br>
+            Myself</h1>
+           <p class="paragraph">I’m a fullstack web developer! I can help you build a <br> website, feature or website Look through some of my<br> work and experience! If you like what you see and have <br> a project you need coded, don’t hestiate to contact me.</p>
+           <h2 class="connect">Let's Connect</h2> 
```

Comment:
- [ ] Kindly correct "LET'S CONNECT" instead of "Let´s Connect" to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193379799-3583e8f3-6ecb-43b7-8938-3cf1ca2c5aaa.png) | ![image](https://user-images.githubusercontent.com/58642949/193379808-fe222f78-4b97-4b7f-bce6-381c28baa6e0.png) |